common/lib/capa/capa/templates/annotationinput.html

```diff
@@ -23,11 +23,23 @@
         <li>
             % if has_options_value:
                 % if all([c == 'correct' for c in option['choice'], status]):
-                <span class="tag-status correct" id="status_${id}" aria-describedby="input_${id}_comment"><span class="sr">Status: Correct</span></span>
+                <span class="tag-status correct" aria-describedby="input_${id}_comment">
+                    <%include file="status_span.html" args="status=status,
+                    classname=status.classname,
+                    status_id=''"/>
+                </span>
                 % elif all([c == 'partially-correct' for c in option['choice'], status]):
-                <span class="tag-status partially-correct" id="status_${id}" aria-describedby="input_${id}_comment"><span class="sr">Status: Partially Correct</span></span>
+                <span class="tag-status partially-correct" aria-describedby="input_${id}_comment">
```

Comment:
Can these if/elif cases be simplified to just a single block of code now, with the 'correct', 'partially-correct', and 'incorrect' just being status.classname?